docs/run-platform-server/monitoring.mdx

```diff
@@ -38,9 +74,34 @@ Here are some example alerts to consider with potential causes and solutions.
 
 ## Logs
 
-Horizon will output logs to standard out. It will log on all aspects of runtime including http requests and ingestion. Typically, there are very few `warn` or `error` severity level messages emitted. The default severity level logged in Horizon is configured to `LOG_LEVEL=info`, this environment configuration parameter can be set to one of `trace, debug, info, warn, error`. The verbosity of log output is inverse of the severity level chosen. I.e. for most verbose logs use 'trace', for least verbose logs use 'error'.
+Horizon will output logs to operating system's standard out. It will log on all aspects of runtime including http requests and ingestion. Typically, there are very few `warn` or `error` severity level messages emitted. The default severity level logged in Horizon is configured to `LOG_LEVEL=info`, this environment configuration parameter can be set to one of `trace, debug, info, warn, error`. The verbosity of log output is inverse of the severity level chosen. I.e. for most verbose logs use 'trace', for least verbose logs use 'error'.
+
+For production deployments, we recommend using the default severity setting of `info` level and choose a log capture strategy depending on the deployment.
+
+- Bare-metal deployment direct to operating system, redirect the standard out from Horizon process to a file on disk and apply a log rotation tool on the file such as [logrotate](https://man7.org/linux/man-pages/man8/logrotate.8.html) to manage disk space usage.
+- Orchestrated deployment on kubernetes, use an EFK/ELK stack on the cluster and it can be configured to capture the standard out from Horizon pod.
+
+## Runtime Profiling
+
+Horizon is written in golang, therefore it has been enabled to optionally emit the golang runtime diagnostics and profiling output (pprof)[https://go.dev/doc/diagnostics]. The pprof http endpoints are hosted on horizon's admin http port, it can be enabled by adding environment configuration parameter `ADMIN_PORT=XXXXX`, since the admin port binding is disabled by default.
```

Comment:
```suggestion
Horizon is written in golang, therefore it has been enabled to optionally emit the golang runtime diagnostics and profiling output [pprof](https://go.dev/doc/diagnostics). The pprof http endpoints are hosted on horizon's admin http port, it can be enabled by adding environment configuration parameter `ADMIN_PORT=XXXXX`, since the admin port binding is disabled by default.
```